core/backend/src/CustomViewState3dCreator.ts

```diff
@@ -61,11 +61,18 @@ export class CustomViewState3dCreator {
     const modelIds = new Set(modelIdsList);
     Logger.logInfo(loggerCategory, "Starting getModelExtents query.");
     for (const id of modelIds) {
+      const modelExtentsStopWatch = new StopWatch("getModelExtents query", false);
       try {
         await new Promise((resolve) => setImmediate(resolve)); // Free up main thread temporarily. Ideally we get queryModelExtents off the main thread and do not need to do this.
+        Logger.logInfo(loggerCategory, "Starting getModelExtents query.");
+        modelExtentsStopWatch.start();
         const props = this._imodel.nativeDb.queryModelExtents({ id }).modelExtents;
+        modelExtentsStopWatch.stop();
+        Logger.logInfo(LoggerCategory, `Finished getModelExtents query. Time taken: ${modelExtentsStopWatch.elapsed} ms.`);
```

Comment:
can we make the elapsed time just log metadata? seems like that will be easier to query than as part of the message...

Reply:
Changed it